### ISSUE TITLE
Unable to recognize SMS verification code in Chinese context

### ISSUE THREAD

**Issue by Mr-Spade** [jd1378/otphelper#1]
Hello, I am a user from China. When using your app, I found that the app cannot recognize the Chinese text message I received containing the verification code.

The specific text message content is:

```
【金山办公】验证码123456，10分钟内有效。验证码提供给他人可能导致账号被盗，请勿转发或泄露。
```

The English translation is:

```
[Kingsoft Office] Verification code 123456, valid within 10 minutes. Providing the verification code to others may lead to account theft, please do not forward or leak it.
```

If you have the energy, could you please see if you can change the recognition strategy to make the application suitable for the Chinese context, I would be very grateful.

الحمد لله :)

**Comment by jd1378** [comment]
Hi
the way app works is it looks for sensitive keywords and then if one is found looks for a code. for example "code" and "otp", so adding one for chinese should do the work, *I guess*
what text do you think if I use as sensitive keyword from the text you provided will cause the app work for nearly all chinese texts ? does `码` work ? do you know of text messages that include `码` generally and do not contain code ?

**Comment by jd1378** [comment]
in a quick test I had, I managed to make the pattern match the code by add `码` as sensitive keyword and a small modification to sentence terminations. 
but I need your response before applying the changes 
thank you

**Comment by Mr-Spade** [comment]
Hello! Thank you for your patience.

I've been looking through my text message history and I think a better keyword would be "验证码".

If you use "码" as the keyword, it will work fine in most cases, but may occasionally include something else. For example, a Chinese text message may contain a "取件码", which usually means giving the number verbally offline to the staff at the delivery station or typing it into a screen on a delivery locker located in the community to be used to pick up your own delivery, rather than typing it into your own cell phone or computer. However, in most cases the additional copying of this information is not an inconvenience.

"验证码" is a good choice in 99% of cases, and only in the very rare case where the text message may be "验证密码", etc., and thus not be recognized. I think it's a good idea to recognize if it contains both "验证" and "码" in this case. Of course, "验证码" itself is almost perfect.

These are just my suggestions and it's up to you to decide what you want to do. Anyway, thank you very much!

**Comment by jd1378** [comment]
Ok so I will use 验证码, because the sensitive keyword can only be used as a single phrase (cannot check for parts separately in the message)
I'll try to release the update tomorrow

**Comment by jd1378** [comment]
released in [v1.2.0](https://github.com/jd1378/otphelper/releases/tag/v1.2.0)
